jwt/__init__.py

```diff
@@ -87,7 +102,12 @@ def encode(payload, key, algorithm='HS256', header=None, encoder=None):
     segments.append(base64url_encode(signature))
     return '.'.join(segments)
 
-def decode(jwt, key='', verify=True):
+def decode(jwt, key='', verify=True, allowed_algos=None):
+    if allowed_algos is not None:
```

Comment:
oh, I see, you added it here. Why not use the same keyword as upstream? ``decode(..., algorithms=None)``. That will make porting easier.